diskfs.go

```diff
@@ -117,9 +117,9 @@ import (
 // when we use a disk image with a GPT, we cannot get the logical sector size from the disk via the kernel
 //    so we use the default sector size of 512, per Rod Smith
 const (
-	defaultBlocksize, firstblock int = 512, 2048
```

Comment:
Is `firstblock` not used anywhere at all?

Reply:
Yes, that is my understanding! I grepped for `firstblock` and it was not used anywhere, and since it's lowercase it seems to be an internal symbol as well. Not sure why it's there! So I decided to remove it as it was confusing.

Reply:
go figure. Thanks for finding it. I am surprised lint didn't catch it.